mne/decoding/time_gen.py

```diff
@@ -390,11 +446,13 @@ def _predict_time_loop(X, estimators, cv, slices, predict_mode,
     ----------
     X : ndarray, shape (n_epochs, n_features, n_times)
         To-be-fitted data.
-    estimators : array-like, shape (n_times, n_folds)
-        Array of scikit-learn classifiers fitted in cross-validation.
-    slices : list
-        List of slices selecting data from X from which is prediction is
-        generated.
+    estimators : list of array-like, shape (n_times, n_folds)
+        List of array of scikit-learn classifiers fitted in cross-validation.
```

Comment:
I agree with jona. I actually never use gat.plot() which I find ugly, and have my own functions for plotting. The tough bit was to make the GAT fast and light and yet accepts any sklearn object.

This shouldn't prevent us to perhaps more clearly separate the high level functions (plotting etc) from the slicer approach (already suggested by Eric a long time ago). If you come up with a plan to make it equally efficient including for cross-condition gen, I'm definitely +1.

The main benefit I see with this, is that if we want to do GAT in time freq space, that'll definely be much easier to have the same slicing approach. ATM, you can do it by making virtual channels for each freq band... Not ideal.